DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
+  name: "John Smith",
+  ID: "300241",                      
+}
+```
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+You can leave out this argument if a template does not have any placeholder fields for personalised information.
 
-##### `phone_number`
+#### reference (optional)
 
-The phone number of the recipient, only required for sms notifications.
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
-##### `template_id`
+```ruby
+reference: "your_reference_string"
+```
 
-Find by clicking **API info** for the template you want to send.
+You can leave out this argument if you do not have a `reference`.
 
-##### `reference`
+#### sms_sender_id (optional)
 
-An optional identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+A unique identifier of the sender of the text message notification. To find this information, go to the __Text Message sender__ settings screen:
 
-You can omit this argument if you do not require a reference for the notification.
+1. Sign in to your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you need to change to another service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the __Text Messages__ section and select __Manage__ on the __Text Message sender__ row.
 
-##### `personalisation`
+In this screen, you can then either:
 
-If a template has placeholders, you need to provide their values, for example:
+  - copy the sender ID that you want to use and paste it into the method
+  - select __Change__ to change the default sender that the service will use, and select __Save__
 
-```python
-personalisation={
-    'first_name': 'Amala',
-    'reference_number': '300241',
-}
+```ruby
+sms_sender_id: "8e222534-7f05-4972-86e3-17c5d9f894e2"
 ```
-##### `sms_sender_id`
 
-Optional. Specifies the identifier of the sms sender to set for the notification. The identifiers are found in your service Settings, when you 'Manage' your 'Text message sender'.
+You can leave out this argument if your service only has one text message sender, or you want to use the default sender.
+
+### Response
+
+If the request to the client is successful, the client will return a `Notifications::Client:ResponseNotification` object. This object is named `smsresponse` in the method example.
 
-If you omit this argument your default sms sender will be set for the notification.
+You can then call different methods on this object to return the requested information:
 
-</details>
+|Method|Information|Type|
+|:---|:---|:---|
+|`smsresponse.id`|Notification UUID|String|
+|`smsresponse.reference`|`reference` argument|String|
+|`smsresponse.content`|- Message body sent to the recipient<br>- SMS sender number of your service|Hash|
```

Comment:
The names of the items that are in the hash are the `body` and the `from_number`. Where we have a hash in the response I think it can be useful to include the names of the items that are in them as well as the description. So perhaps something along the lines of

- `body` Message body sent to the recipient
- `from_number` SMS sender number of your service